core/services/feeds/service.go

```diff
@@ -510,6 +510,21 @@ func (s *service) DeleteJob(ctx context.Context, args *DeleteJobArgs) (int64, er
 		logger.Errorw("Failed to push metrics for job proposal deletion", "err", err)
 	}
 
+	job, err := s.jobORM.FindJobByExternalJobID(ctx, proposal.ExternalJobID.UUID)
+	if err != nil {
```

Comment:
```suggestion
	if !errors.Is(err, sql.ErrNoRows) {
```

Reply:
oh yeah just copy the blob that @giogam linked :D including the proposal.ExternalJobID.Valid check since that would return false when the job is not in approved status.

Reply:
Why should we treat the NoRows error in a special way? Why not just catch any error?

Reply:
Typically it is the `ORM`'s responsibility to hide the sql details by catching this error internally and returning `nil` instead.

Reply:
hmm I'm confused - is my code going to work or not? If there's any error, I give up and don't delete. That should be fine if nothing is found. What else can go wrong?

Reply:
Oh I see it just logs it and doesn't return, and with the .valid check before that, it looks like it should be fine now